src/components/authenticatedComponents/MobileNavbarAuth.tsx

```diff
@@ -35,7 +35,7 @@ const MobileNavbarAuth = () => {
         Profile
         </Link>
 
-        <Link to="/projects" className="mobileLink subLink" onClick={handleMenuClick}>
+        <Link to="/addInvestments" className="mobileLink subLink" onClick={handleMenuClick}>
```

Comment:
can you remind me about sorting the navigation out when we have all the pages worked out and running pls :)